sync/p2p_messages.md

```diff
@@ -124,6 +126,14 @@ hash, see `aec_peer_messages` for details.
 Message is RLP encoded, fields:
   - `Hash :: byte_array`
 
+## MSG_TXS
+*(Tag = 9)*
+
+Message is RLP encoded, fields:
+  - `Txs:: [byte_array]`
+
+A signed transaction is serialized using the `aetx_sign:serialize_to_binary/1`.
```

Comment:
Maybe prefer refer to spec rather than code? E.g. "... is serialized as a tagged and versioned [signed transaction](/serializations.md#signed-transaction)"